### PR TITLE
fix(Prometheus): Fixing some errors, urls, typos, etc.

### PR DESCRIPTION
**Pull request by nbaenam**
Fixed these requests coming from the help-documentation channel (@mangulonr):

- Add these 3 new pages from [this PR](https://github.com/newrelic/docs-website/pull/11156) from yesterday to the left menu
- Include them also in the [Prometheus integrations list](https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/integrations-list/integrations-list-intro/)
- Fix the [Harbor](https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/integrations-list/harbor-integration/) document format, it looks like the format is broken.

### CODE REVIEW

**Review by mangulonr — APPROVED** [review]

File: src/content/docs/infrastructure/prometheus-integrations/troubleshooting/no-data-appears-prometheus-integration.mdx [hunk 1]
```diff
@@ -48,7 +48,7 @@ Follow these troubleshooting tips for Docker or Kubernetes as applicable:
 
     4. Run this NRQL query:
 
-       ```
+       ```sql
        docker logs nri-prometheus | grep "error emitting metrics"
```
Comment (x8a) [inline src/content/docs/infrastructure/prometheus-integrations/troubleshooting/no-data-appears-prometheus-integration.mdx:52]: is this a NRQL query or a docker command?

File: src/content/docs/infrastructure/prometheus-integrations/troubleshooting/no-data-appears-prometheus-integration.mdx [hunk 2]
```diff
@@ -26,9 +26,9 @@ Follow these troubleshooting tips for Docker or Kubernetes as applicable:
 <CollapserGroup>
   <Collapser
     id="docker"
-    title={<><img src={osDocker} title="Docker icon" alt="Docker icon"/> Docker troubleshooting</>}
+    title={<><img style={{ width: '32px', height: '28px', verticalAlign: 'middle'}} class="inline" title="Docker" alt="Docker" src={osDocker}/> Docker troubleshooting</>}
   >
-    If you are having problems with the integration:
+    If you're having problems with the integration:
 
     1. Check if the Prometheus OpenMetrics integration is running:
```
Comment (x8a) [inline src/content/docs/infrastructure/prometheus-integrations/troubleshooting/no-data-appears-prometheus-integration.mdx:34]: we could add `shell` in the code snippets to color them.

File: src/content/docs/infrastructure/prometheus-integrations/integrations-list/argocd-integration.mdx [hunk 3]
```diff
@@ -10,9 +10,9 @@ tags:
 
 import kubernetesArgoCDDashboard from 'images/kubernetes_screenshot-crop_argocd-dashboard.png'
 
-[Argo CD](https://argoproj.github.io/) Argo CD is a Kubernetes controller, responsible for continuously monitoring all running applications and comparing their live state to the desired state specified in the Git repository.
+[Argo CD](https://argoproj.github.io/) is a Kubernetes controller, responsible for continuously monitoring all running applications and comparing their live state to the desired state specified in the Git repository.
 
-Use New Relic to view a dashboard based on Prometheus metrics which helps you understand the Argo CD infrastructure of your k8s cluster. With New Relic you can monitor: 
+Use New Relic to view a dashboard based on Prometheus metrics. It helps you understand the Argo CD infrastructure of your k8s cluster. With New Relic you can monitor:
```
Comment (x8a) [inline src/content/docs/infrastructure/prometheus-integrations/integrations-list/argocd-integration.mdx:15]:
```suggestion
Use New Relic to view a dashboard based on Prometheus metrics to better understand the Argo CD infrastructure of your k8s cluster. 
```

**Review by x8a — DISMISSED** [review]
I have some comments, but apart from those, everything looks good 👌🏻!

File: src/content/docs/infrastructure/prometheus-integrations/troubleshooting/no-data-appears-prometheus-integration.mdx (hunk 1 above)
Comment (nbaenam) [reply src/content/docs/infrastructure/prometheus-integrations/troubleshooting/no-data-appears-prometheus-integration.mdx:52]: Good catch!

File: src/content/docs/infrastructure/prometheus-integrations/integrations-list/argocd-integration.mdx (hunk 3 above)
Comment (nbaenam) [reply src/content/docs/infrastructure/prometheus-integrations/integrations-list/argocd-integration.mdx:15]: Thanks!

**Review by x8a — APPROVED** [review]

**Pull request merged**